solutions/instances/main.tf

```diff
@@ -121,16 +178,50 @@ module "cos" {
       metrics_monitoring_crn  = var.existing_monitoring_crn
     }
   }]
+}
+
+module "cos" {
+  providers = {
+    ibm = ibm.cos
+  }
+  count                    = var.existing_scc_cos_bucket_name == null && var.existing_scc_instance_crn == null ? 1 : 0 # no need to call COS module if consumer is passing existing SCC instance or COS bucket
+  source                   = "terraform-ibm-modules/cos/ibm//modules/fscloud"
+  version                  = "8.11.14"
+  resource_group_id        = module.resource_group.resource_group_id
+  create_cos_instance      = var.existing_cos_instance_crn == null ? true : false # don't create instance if existing one passed in
+  cos_instance_name        = local.cos_instance_name
+  cos_tags                 = var.cos_instance_tags
+  existing_cos_instance_id = var.existing_cos_instance_crn
+  access_tags              = var.cos_instance_access_tags
+  cos_plan                 = "standard"
+  bucket_configs           = local.create_cross_account_auth_policy ? [] : local.bucket_config
+}
 
+module "buckets" {
```

Comment:
please leave code comments as to why we have to do this outside of the cos module call (aka for cross account auth policy purposes)